src/contrast.rs

```diff
@@ -99,6 +99,22 @@ pub fn otsu_level(image: &GrayImage) -> u8 {
     best_threshold
 }
 
+/// type of the threshold operation, corresponding to opencv threshold type
+///
+/// `<https://docs.opencv.org/4.x/d7/d1b/group__imgproc__misc.html#gaa9e58d2860d4afa658ef70a9b1115576>`
+pub enum ThresholdType {
+    /// `dst(x,y) = maxval if src(x,y) > thresh else 0`
+    ThreshBinary,
```

Comment:
As the type name already includes “Threshold”, let’s just called the variants “Binary”, “BinaryInverse”, etc.